java/code/src/com/redhat/rhn/frontend/strings/template/StringResource_en_US.xml

```diff
@@ -174,11 +174,6 @@ Summary:
 ---------------------
 Affected Systems List
 ---------------------
-This Patch Advisory may apply to the systems listed below. If you know that
-this patch does not apply to a system listed, it might be possible that the
-package profile for that server is out of date. In that case you should run
-'up2date -p' (RHEL 4 and below) or 'rhn-profile-sync' (SLES, RHEL 5 and above)
-as root on the system in question to refresh your software profile.
```

Comment:
I think not everything is obsolete. The first 2 sentence are still correct and important.